src/components/Page/Footer.tsx

```diff
@@ -0,0 +1,71 @@
+import classNames from 'classnames';
```

Comment:
`Footer` is not a good name for this component. Reread [Task checklist](https://github.com/mate-academy/react_todo-app-loading-todos/blob/master/checklist.md). 

Be always attentive with tasks descriptions and checklists. There **no redundant** words. The same would be in test tasks. You need to read it attentively and write the code according to it. It is important.